__tests__/app.test.js

```diff
@@ -311,3 +311,30 @@ describe("/api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  test("GET 200 Responds with an array of objects containing all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((user) => {
```

Comment:
don't forget you need a length check here to prevent a false positive on this test where the array is empty